app/src/main/java/org/jellyfin/androidtv/ui/browsing/BrowseGridFragment.java

```diff
@@ -719,8 +719,7 @@ public void onResponse() {
                     updateCounter(mAdapter.getTotalItems() > 0 ? 1 : 0);
                 }
                 mLetterButton.setVisibility(ItemSortBy.SortName.equals(mAdapter.getSortBy()) ? View.VISIBLE : View.GONE);
-                if (mAdapter.getTotalItems() == 0) {
-                    binding.toolBar.requestFocus();
```

Comment:
Wait, really? I won't have access to a computer for a couple of days so I can't really test it, but if I recall correctly the focus stays on the button with no problems.

I think I removed that line because it caused the focus to move to the first button in the list which was kinda jank.

Reply:
I tested it with an empty library (Live TV recordings), when I open it before this PR it immediately switches focus to the toolbar, with the PR it does not.

Reply:
Hmm... Ok, I'll see what I can do.